wallets/provider-clover/src/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { type ProviderInfo } from '@rango-dev/wallets-core';
+
+export const WALLET_ID = 'clover';
+
+export const info: ProviderInfo = {
+  name: 'Clover',
```

Comment:
I think they rebranded to CLV, wouldn't it better to use that name?

Reply:
I changed it.